README.md

```diff
@@ -3,14 +3,19 @@
 [![Go](https://github.com/alibaba/sealer/actions/workflows/go.yml/badge.svg)](https://github.com/alibaba/sealer/actions/workflows/go.yml)
 [![Release](https://github.com/alibaba/sealer/actions/workflows/release.yml/badge.svg)](https://github.com/alibaba/sealer/actions/workflows/release.yml)
 
-# What is CloudImage 
+# What is sealer
 
-![image](https://user-images.githubusercontent.com/8912557/117261089-5467c380-ae82-11eb-8dd8-1163c1a74b10.png)
+sealer[ˈsiːlər] is a solution for distributed application package and delivery base on kubernetes. 
```

Comment:
Word solution may mislead others that sealer is not a general way.

s/base/based

🐸